Library/Formula/metashell.rb

```diff
@@ -10,20 +11,28 @@ class Metashell < Formula
     sha1 "76874d93ced1815ccfcbddb6da75e8c56aff44e4" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
-
   needs :cxx11
 
+  # This patch is required because Mountain Lion uses an old compiler which breaks
+  # compiling some Templight code. The patch comments out unused parts of Templight,
+  # so patched version is functionally equivalent. This error should be fixed in
+  # the next release of Metashell.
+  patch :DATA if MacOS.version == :mountain_lion
```

Comment:
Latest GCC, really?

The command line in jenkins shows that `/usr/local/Library/ENV/4.3/clang++` were running. The version is according to the environment variables is `Clang: 5.1 build 503`, which is some Apple version of clang 3.4 according to this https://gist.github.com/yamaya/2924292 .